data/authors/vctqs1.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "vctqs1",
+    "bio": "I'm just someone who's willing to learn, according to everything automation work, and be impressed with what looks good",
```

Comment:
"bio": "I'm just a normal person, who's willing to learn, trying to automate everything, and impressed by what looks good",

Reply:
ok